lib/schema/src/openctx.schema.ts

```diff
@@ -1,18 +1,66 @@
 /**
  * Metadata about code
  */
-export type Schema = Item | Annotation
+export type Schema = CandidateItem | Item | Annotation
```

Comment:
need to edit the `*.schema.json` files not these auto-generated files

Reply:
Done.